package.json

```diff
@@ -46,11 +48,13 @@
     "gruntfile-editor": "^0.2.0",
     "inquirer": "^0.8.0",
     "isbinaryfile": "^2.0.0",
+    "istextorbinary": "^1.0.2",
```

Comment:
Currently, `isbinaryfile` is loads 512 first bytes in a buffer, and then make the check with this buffer. `istextorbinary` has a buffer as input, so it's up to the user to load the buffer, and it then reads the buffer until it reach a binary byte.

So it seems that using only 512 bytes of data could be enough, and would make `isbinaryfile` and `istextorbinary` quite equivalent ...